### ISSUE TITLE
"Resize" option is not provided for adjusting column header: A11y_MSSQL for VSCode_Add Connection_Dragging Movements

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/vscode-mssql#19079]
**Github Tags:** #A11yTCS; #A11ySev3; #A11yMAS; #WCAG2.5.7; #DraggingMovements; #FTP; #Win32; #Win11; #DesktopApp; #MSSQL for VSCode; #BM_MSSQL for VSCode_Win32_Feb2025; #A11yWCAG2.2;

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Environment Details:

Application Name: MSSQL for VSCode
Windows 11 24H2 OS Build 26100.3037

## Repro Steps:

1. Launch VS Code Application.
2. Open SQL Server Extension.
3. Add a valid Connection.
4. Tab till the database and hit enter.
5. Tab till the System database and hit enter.
6. Tab till the Tables and hit enter.
7. Tab till any table under System tables and hit shift + F10
8. Tab till Select Top 1000 and hit enter
9. Tab till Execute Query and hit enter
10. Tab till "Results" dropdown and hit enter
11. Tab till grid and verify whether column width is adjustable or not.

 
## Actual Results: 
"Resize" option is not provided for adjusting column header present in case grid.

## Expected Results:
The column width should be adjustable the column headers by using the resize option, that users can resize the column to fit their needs and preferences. This allows for better accessibility and usability.

## User Impact:
people with motor impairments who cannot operate content because the only way to actuate a function is by dragging a target element from its initial position to some other position.

## WCAG Reference:  

https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements

## Attachments:

**Comment by laurenastrid1** [comment]
Please use the latest version of the VS Code extension. This UI is no longer present in the extension.

**Comment by laurenastrid1** [comment]
Closing because this UI no longer exists
